### PR TITLE
Feature/screen orientation

### PR DESCRIPTION
**Pull request by dmiales**
Continued: https://github.com/home-assistant/android/pull/3352

This PR adds a setting to fix the app's landscape/portrait/system orientation. This is extremely convenient for lovelace (yaml) display when we do not want to create a separate version of lovelace for mobile devices.
Example.
Looks awful in portrait orientation:

And in the landscape is perfect:

We fix the landscape orientation in the settings
изображение
изображение

And the user gets a beautiful board

### CODE REVIEW

File: app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt [hunk 1]
```diff
@@ -744,6 +745,13 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
 
         WebView.setWebContentsDebuggingEnabled(BuildConfig.DEBUG || presenter.isWebViewDebugEnabled())
 
+        requestedOrientation = if (presenter.getScreenOrientation() == applicationContext.getString(commonR.string.screen_orientation_option_value_portrait))
```
[the remark this answers is not in the document]
Comment (dmiales) [reply app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt]: done

File: app/src/main/res/xml/preferences.xml [hunk 2]
```diff
@@ -39,6 +39,14 @@
             android:icon="@drawable/ic_fullscreen"
             android:title="@string/fullscreen"
             android:summary="@string/fullscreen_def"/>
+        <ListPreference
+            android:key="screen_orientation"
+            android:defaultValue="@string/screen_orientation_option_value_system"
+            android:entries="@array/pref_screen_orientation_option_labels"
+            android:entryValues="@array/pref_screen_orientation_option_values"
+            android:icon="@drawable/ic_landscape"
+            android:summary="@string/screen_orientation_def"
```
[the remark this answers is not in the document]
Comment (dmiales) [reply app/src/main/res/xml/preferences.xml]: done

**Comment by dmiales** [comment]
изображение

File: common/src/main/res/drawable/ic_landscape.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
[the remark this answers is not in the document]
Comment (jpelgrom) [reply common/src/main/res/drawable/ic_landscape.xml]: It also can be moved to the `app` module resources, as it isn't used in the Wear app

**Review by jpelgrom — APPROVED** [review]
Works as described, looks good to me

**Comment by dshokouhi** [comment]
@dmiales sorry I totally forgot to ask but would you be able to submit a docs PR to document this feature over on https://companion.home-assistant.io/docs/integrations/android-webview ?  Just needs a beta label added to it, you can see in previous PR's the proper tag to use.

**Comment by dmiales** [comment]
@dshokouhi https://github.com/home-assistant/companion.home-assistant/pull/917

**Pull request merged**